dev/com.ibm.ws.security.kerberos.auth/resources/OSGI-INF/metatype/metatype.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+    Copyright (c) 2020 IBM Corporation and others.
+    All rights reserved. This program and the accompanying materials
+    are made available under the terms of the Eclipse Public License v1.0
+    which accompanies this distribution, and is available at
+    http://www.eclipse.org/legal/epl-v10.html
+
+    Contributors:
+        IBM Corporation - initial API and implementation
+-->
+<metatype:MetaData xmlns:metatype="http://www.osgi.org/xmlns/metatype/v1.1.0" 
+                   xmlns:ibm="http://www.ibm.com/xmlns/appservers/osgi/metatype/v1.0.0" 
+                   localization="OSGI-INF/l10n/metatype">
+    
+    <OCD id="com.ibm.ws.security.kerberos.auth.KerberosService" name="%kerberos" description="%kerberos.desc" 
+    	  ibm:alias="kerberos" ibm:beta="true">
+    	  
+    	<AD id="keytab"     name="%kerberos.keytab"     description="%kerberos.keytab.desc"     type="String" ibm:type="location(file)" required="false"/>
```

Comment:
Can we remove Kerberos from name and description?
<AD id="keytab"     name="%keytab"     description="%keytab.desc"     type="String" ibm:type="location(file)" required="false"/>
    	<AD id="configFile" name="%configFile" description="%configFile.desc" type="String" ibm:type="location(file)" required="false"/>

Reply:
those are just variable names, so a user would not seem them. I can shorten the variable names though if that's what you're asking

Reply:
Yes, just shorting the variable names.. %keytab and %keytab.des